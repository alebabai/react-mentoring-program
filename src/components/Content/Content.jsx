import React from 'react'

import ResultsViewer from '../ResultsViewer'

const mockedData = [{
    id: '1',
    imageUrl: null,
    title: 'Fight Club',
    genre: 'Action',
    releaseDate: '1995'
}]

export const Content = () => (
    <article className="content">
        <ResultsViewer items={mockedData}/>
    </article>
)

export default Content
