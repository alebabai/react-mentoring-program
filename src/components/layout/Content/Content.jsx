import React from 'react'

export const Content = ({ children }) => (
    <article className="content">
        {children}
    </article>
)

export default Content
