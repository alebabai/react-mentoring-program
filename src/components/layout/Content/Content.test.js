import React from 'react'

import Content from './Content'

import renderer from 'react-test-renderer';

describe('Should match snapshot', () => {
    test('empty', () => {
        const component = renderer.create(
            <Content />
        )
        expect(component.toJSON()).toMatchSnapshot()
    })

    test('with children', () => {
        const component = renderer.create(
            <Content>
                <div className="wrapper-1">
                    <h1>Some title 1</h1>
                </div>
                <div className="wrapper-2">
                    <h1>Some title 2</h1>
                </div>
            </Content>
        )
        expect(component.toJSON()).toMatchSnapshot()
    })
})

