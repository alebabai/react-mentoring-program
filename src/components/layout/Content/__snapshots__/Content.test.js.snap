// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should match snapshot empty 1`] = `
<article
  className="content"
/>
`;

exports[`Should match snapshot with children 1`] = `
<article
  className="content"
>
  <div
    className="wrapper-1"
  >
    <h1>
      Some title 1
    </h1>
  </div>
  <div
    className="wrapper-2"
  >
    <h1>
      Some title 2
    </h1>
  </div>
</article>
`;
